api/api_test.go

```diff
@@ -161,7 +161,7 @@ func TestSetQueryOptions(t *testing.T) {
 	c, s := makeClient(t, nil, nil)
 	defer s.Stop()
 
-	r, _ := c.newRequest("GET", "/v1/jobs")
+	r, _ := c.newRequest("GET", "/v1/jobs?format=baz")
```

Comment:
Note to reviewers: this test change only exists for me to verify that we avoid overwriting query params with `QueryOptions`, and I left it in to document the behavior.